autograd.h

```diff
@@ -475,7 +475,9 @@ AUTOGRAD_OPTIMIZER_CLASS(Adam) {
   std::unordered_map<std::string, at::Tensor> exp_avg_sq_buffer_;
 };
 
-}  // namespace autograd
+void setSeed(int seed);
```

Comment:
Too late, by why not have this take a `uint64_t`?

Reply:
#36